### DISCUSSION TITLE
Shall we use boldface to emphasize text?

### DISCUSSION THREAD

**Discussion by HeatherWYL** [AgoraDoc/agora-style-guide#26] [General] [answered]
In [Set the Subscribing State](https://confluence.agoralab.co/display/TEKP/Set+the+Subscribing+State), I want to highlight the differences between different SDK versions, so I use the boldface in the following sentences:
mute-related methods can be called **both before and after** joining a channel or switching to a channel.
Call mute-related methods **after** joining a channel or switching to a channel; otherwise these methods do not take effect.

Shall we use boldface to emphasize text?

**Comment by Rebeccacy** [comment, marked as answer]
I think it's acceptable. But we should not overuse it.
